src/styles/theme/helper.scss

```diff
@@ -3,18 +3,17 @@
 }
 
 // Directions
-$directions: "top", "right", "bottom", "left";
+$directions: 'top', 'right', 'bottom', 'left';
 
 .oc-cursor-pointer {
   cursor: pointer;
 }
 
 .oc-button-reset {
-  padding: 0;
-  -webkit-appearance: none;
   background: transparent;
+  border: 0;
   font: inherit;
-  border: none;
+  padding: 0;
```

Comment:
It's mainly because `.uk-padding-remove` uses `!important`. This way it's gonna be easier to give the button different padding if needed. Getting rid of UIKit in yet another place in the code is "just" a nice bonus.